scripts/lib/scss-vars.mjs

```
@@ -0,0 +1,337 @@
+ * Keys registered via `@include cv.register-vars((...))`, at any nesting depth.
+ *
+ * Kept deliberately identical to the original implementation in
+ * check-conformance.mjs (plural form only, all depths) — `scss-conformance` and
+ * `skills-sync` are ratcheted against this exact behaviour, and widening it
+ * would newly fail partials that were already passing.
+ */
+export function registerVarsKeys(src) {
+  const keys = new Set();
+  const re = /cv\.register-vars\s*\(/g;
+  let m;
+  while ((m = re.exec(src))) {
```

ESLint

[error] 'm' is assigned a value but never used.

(no-unused-vars)